### PR TITLE
 [Sprint 22/23 | PD-421] - [Enhancement] Use Jitsuyo For Utilities

### PR DESCRIPTION
**Pull request by Mobilizes**
## Jira Link: 
https://ichiro-its.atlassian.net/jira/software/c/projects/PD/boards/4?selectedIssue=PD-421

## Description
Use jitsuyo repository to load and save config

## Type of Change

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause the existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] New unit tests added.
- [ ] Manual tested.

## Checklist:

- [ ] Using Branch Name Convention
    - `feature/JIRA-ID-SHORT-DESCRIPTION` if has a JIRA ticket
    - `enhancement/SHORT-DESCRIPTION` if has/has no JIRA ticket and contain enhancement
    - `hotfix/SHORT-DESCRIPTION` if the change doesn't need to be tested (urgent)
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made the documentation for the corresponding changes.

### CODE REVIEW

File: include/shisen_cpp/camera/node/camera_node.hpp [hunk 1]
```diff
@@ -21,10 +21,10 @@
 #ifndef SHISEN_CPP__CAMERA__NODE__CAMERA_NODE_HPP_
 #define SHISEN_CPP__CAMERA__NODE__CAMERA_NODE_HPP_
 
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/camera/provider/camera_config_provider.hpp>
-#include <shisen_cpp/camera/provider/image_provider.hpp>
-#include <shisen_cpp/utility.hpp>
+#include "rclcpp/rclcpp.hpp"
```
Comment (threeal) [inline include/shisen_cpp/camera/node/camera_node.hpp]: https://www.learncpp.com/cpp-tutorial/header-files/#includemethod

```suggestion
#include <rclcpp/rclcpp.hpp>
```

File: include/shisen_cpp/camera/provider/camera_config_provider.hpp [hunk 2]
```diff
@@ -21,9 +21,9 @@
 #ifndef SHISEN_CPP__CAMERA__PROVIDER__CAMERA_CONFIG_PROVIDER_HPP_
 #define SHISEN_CPP__CAMERA__PROVIDER__CAMERA_CONFIG_PROVIDER_HPP_
 
-#include <keisan/keisan.hpp>
-#include <shisen_cpp/utility/options.hpp>
-#include <shisen_interfaces/msg/camera_config.hpp>
+#include "keisan/keisan.hpp"
```
Comment (threeal) [inline include/shisen_cpp/camera/provider/camera_config_provider.hpp]: Ditto.
```suggestion
#include <keisan/keisan.hpp>
```

File: include/shisen_cpp/camera/provider/image_provider.hpp [hunk 3]
```diff
@@ -21,12 +21,12 @@
...
-#include <opencv2/videoio.hpp>
-#include <sensor_msgs/msg/image.hpp>
-#include <shisen_cpp/utility.hpp>
-#include <std_msgs/msg/header.hpp>
+#include "cv_bridge/cv_bridge.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/videoio.hpp"
+#include "sensor_msgs/msg/image.hpp"
+#include "shisen_cpp/utility.hpp"
+#include "std_msgs/msg/header.hpp"
```
Comment (threeal) [inline include/shisen_cpp/camera/provider/image_provider.hpp]: Ditto.

File: include/shisen_cpp/config/grpc/call_data.hpp [hunk 4]
```diff
@@ -21,9 +21,9 @@
 #ifndef __SHISEN_CPP__CONFIG__GRPC__CALL_DATA_HPP__
 #define __SHISEN_CPP__CONFIG__GRPC__CALL_DATA_HPP__
 
-#include <shisen_cpp/config/grpc/call_data_base.hpp>
-#include <shisen_interfaces/shisen.grpc.pb.h>
-#include <shisen_interfaces/shisen.pb.h>
+#include "shisen_cpp/config/grpc/call_data_base.hpp"
+#include "shisen_interfaces/shisen.grpc.pb.h"
+#include "shisen_interfaces/shisen.pb.h"
```
Comment (threeal) [inline include/shisen_cpp/config/grpc/call_data.hpp]: Ditto.

File: include/shisen_cpp/config/grpc/config.hpp [hunk 5]
```diff
@@ -21,10 +21,10 @@
...
 #define SHISEN_CPP__CONFIG__GRPC__CONFIG_HPP_
 
-#include <grpcpp/grpcpp.h>
-#include <shisen_cpp/camera/node/camera_node.hpp>
-#include <shisen_interfaces/shisen.grpc.pb.h>
-#include <shisen_interfaces/shisen.pb.h>
+#include "grpcpp/grpcpp.h"
+#include "shisen_cpp/camera/node/camera_node.hpp"
+#include "shisen_interfaces/shisen.grpc.pb.h"
+#include "shisen_interfaces/shisen.pb.h"
```
Comment (threeal) [inline include/shisen_cpp/config/grpc/config.hpp]: Ditto.

File: include/shisen_cpp/node/shisen_cpp_node.hpp [hunk 6]
```diff
@@ -21,11 +21,10 @@
 #ifndef SHISEN_CPP__NODE__SHISEN_CPP_NODE_HPP_
 #define SHISEN_CPP__NODE__SHISEN_CPP_NODE_HPP_
 
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/camera/node/camera_node.hpp>
-#include <shisen_cpp/config/grpc/config.hpp>
-#include <shisen_cpp/config/utils/config.hpp>
-#include <shisen_cpp/utility.hpp>
+#include "rclcpp/rclcpp.hpp"
```
Comment (threeal) [inline include/shisen_cpp/node/shisen_cpp_node.hpp]: Ditto.

File: include/shisen_cpp/utility/capture_setting.hpp [hunk 7]
```diff
@@ -21,8 +21,8 @@
 #ifndef SHISEN_CPP__UTILITY__CAPTURE_SETTING_HPP_
 #define SHISEN_CPP__UTILITY__CAPTURE_SETTING_HPP_
 
-#include <shisen_cpp/utility/emptiable.hpp>
-#include <shisen_interfaces/msg/capture_setting.hpp>
+#include "shisen_cpp/utility/emptiable.hpp"
+#include "shisen_interfaces/msg/capture_setting.hpp"
```
Comment (threeal) [inline include/shisen_cpp/utility/capture_setting.hpp]: Ditto.

File: include/shisen_cpp/utility/interface.hpp [hunk 8]
```diff
@@ -21,9 +21,9 @@
 #ifndef SHISEN_CPP__UTILITY__INTERFACE_HPP_
 #define SHISEN_CPP__UTILITY__INTERFACE_HPP_
 
-#include <sensor_msgs/msg/image.hpp>
-#include <shisen_interfaces/msg/camera_config.hpp>
-#include <shisen_interfaces/srv/configure_capture_setting.hpp>
+#include "sensor_msgs/msg/image.hpp"
+#include "shisen_interfaces/msg/camera_config.hpp"
+#include "shisen_interfaces/srv/configure_capture_setting.hpp"
```
Comment (threeal) [inline include/shisen_cpp/utility/interface.hpp]: Ditto.

File: include/shisen_cpp/viewer/consumer/image_consumer.hpp [hunk 9]
```diff
@@ -21,11 +21,11 @@
...
 
-#include <cv_bridge/cv_bridge.hpp>
-#include <opencv2/core.hpp>
-#include <opencv2/highgui.hpp>
-#include <sensor_msgs/msg/image.hpp>
-#include <shisen_cpp/utility.hpp>
+#include "cv_bridge/cv_bridge.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/highgui.hpp"
+#include "sensor_msgs/msg/image.hpp"
```
Comment (threeal) [inline include/shisen_cpp/viewer/consumer/image_consumer.hpp]: Ditto.

File: include/shisen_cpp/viewer/node/viewer_node.hpp [hunk 10]
```diff
@@ -21,10 +21,10 @@
 #ifndef SHISEN_CPP__VIEWER__NODE__VIEWER_NODE_HPP_
 #define SHISEN_CPP__VIEWER__NODE__VIEWER_NODE_HPP_
 
-#include <rclcpp/rclcpp.hpp>
-#include <sensor_msgs/msg/image.hpp>
-#include <shisen_cpp/utility.hpp>
-#include <shisen_cpp/viewer/consumer/image_consumer.hpp>
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/image.hpp"
```
Comment (threeal) [inline include/shisen_cpp/viewer/node/viewer_node.hpp]: Ditto.

File: src/shisen_cpp/camera/node/camera_node.cpp [hunk 11]
```diff
@@ -18,12 +18,9 @@
...
 // THE SOFTWARE.
 
-#include <fstream>
-#include <nlohmann/json.hpp>
-#include <fstream>
-#include <memory>
-#include <nlohmann/json.hpp>
-#include <shisen_cpp/camera/node/camera_node.hpp>
+#include "jitsuyo/config.hpp"
+#include "nlohmann/json.hpp"
```
Comment (threeal) [inline src/shisen_cpp/camera/node/camera_node.cpp]: Ditto.

File: src/shisen_cpp/camera/provider/image_provider.cpp [hunk 12]
```diff
@@ -18,10 +18,10 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#include <shisen_cpp/camera/provider/image_provider.hpp>
+#include "shisen_cpp/camera/provider/image_provider.hpp"
+#include "opencv2/opencv.hpp"
```
Comment (threeal) [inline src/shisen_cpp/camera/provider/image_provider.cpp]: Ditto.

File: src/shisen_cpp/config/grpc/call_data_get_capture_setting.cpp [hunk 13]
```diff
@@ -18,11 +18,11 @@
...
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/config/grpc/call_data_get_capture_setting.hpp>
-#include <shisen_cpp/config/utils/config.hpp>
-#include <shisen_interfaces/shisen.grpc.pb.h>
-#include <shisen_interfaces/shisen.pb.h>
+#include "jitsuyo/config.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "shisen_cpp/config/grpc/call_data_get_capture_setting.hpp"
+#include "shisen_interfaces/shisen.grpc.pb.h"
+#include "shisen_interfaces/shisen.pb.h"
```
Comment (threeal) [inline src/shisen_cpp/config/grpc/call_data_get_capture_setting.cpp]: Ditto.

File: src/shisen_cpp/config/grpc/call_data_get_image.cpp [hunk 14]
```diff
@@ -18,11 +18,11 @@
...
-#include <opencv2/opencv.hpp>
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/config/grpc/call_data_get_image.hpp>
-#include <shisen_interfaces/shisen.grpc.pb.h>
-#include <shisen_interfaces/shisen.pb.h>
+#include "opencv2/opencv.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "shisen_cpp/config/grpc/call_data_get_image.hpp"
+#include "shisen_interfaces/shisen.grpc.pb.h"
+#include "shisen_interfaces/shisen.pb.h"
```
Comment (threeal) [inline src/shisen_cpp/config/grpc/call_data_get_image.cpp]: Ditto.

File: src/shisen_cpp/config/grpc/call_data_save_capture_setting.cpp [hunk 15]
```diff
@@ -18,12 +18,12 @@
...
-#include <shisen_cpp/config/grpc/call_data_save_capture_setting.hpp>
-#include <shisen_cpp/config/utils/config.hpp>
-#include <shisen_interfaces/shisen.grpc.pb.h>
-#include <shisen_interfaces/shisen.pb.h>
+#include "jitsuyo/config.hpp"
+#include "nlohmann/json.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "shisen_cpp/config/grpc/call_data_save_capture_setting.hpp"
+#include "shisen_interfaces/shisen.grpc.pb.h"
+#include "shisen_interfaces/shisen.pb.h"
```
Comment (threeal) [inline src/shisen_cpp/config/grpc/call_data_save_capture_setting.cpp]: Ditto.

File: src/shisen_cpp/config/grpc/call_data_set_capture_setting.cpp [hunk 16]
```diff
@@ -18,11 +18,11 @@
...
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/config/grpc/call_data_set_capture_setting.hpp>
-#include <shisen_cpp/config/utils/config.hpp>
-#include <shisen_interfaces/shisen.grpc.pb.h>
-#include <shisen_interfaces/shisen.pb.h>
+#include "nlohmann/json.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "shisen_cpp/config/grpc/call_data_set_capture_setting.hpp"
+#include "shisen_interfaces/shisen.grpc.pb.h"
+#include "shisen_interfaces/shisen.pb.h"
```
Comment (threeal) [inline src/shisen_cpp/config/grpc/call_data_set_capture_setting.cpp]: Ditto.

File: src/shisen_cpp/config/grpc/config.cpp [hunk 17]
```diff
@@ -18,14 +18,14 @@
...
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/config/grpc/call_data_base.hpp>
-#include <shisen_cpp/config/grpc/call_data_get_capture_setting.hpp>
-#include <shisen_cpp/config/grpc/call_data_get_image.hpp>
-#include <shisen_cpp/config/grpc/call_data_save_capture_setting.hpp>
-#include <shisen_cpp/config/grpc/call_data_set_capture_setting.hpp>
-#include <shisen_cpp/config/grpc/config.hpp>
-#include <shisen_cpp/config/utils/config.hpp>
+#include "jitsuyo/config.hpp"
+#include "rclcpp/rclcpp.hpp"
```
Comment (threeal) [inline src/shisen_cpp/config/grpc/config.cpp]: Ditto.

File: src/shisen_cpp_main.cpp [hunk 18]
```diff
@@ -18,8 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/shisen_cpp.hpp>
+#include "rclcpp/rclcpp.hpp"
```
Comment (threeal) [inline src/shisen_cpp_main.cpp]: Ditto.

File: src/viewer_main.cpp [hunk 19]
```diff
@@ -18,8 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/shisen_cpp.hpp>
+#include "rclcpp/rclcpp.hpp"
```
Comment (threeal) [inline src/viewer_main.cpp]: Ditto.

File: test/capture_setting_test.cpp [hunk 20]
```diff
@@ -18,8 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#include <gtest/gtest.h>
-#include <shisen_cpp/shisen_cpp.hpp>
+#include "gtest/gtest.h"
```
Comment (threeal) [inline test/capture_setting_test.cpp]: Ditto.

File: test/compile_test.cpp [hunk 21]
```diff
@@ -18,9 +18,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#include <gtest/gtest.h>
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/shisen_cpp.hpp>
+#include "gtest/gtest.h"
+#include "rclcpp/rclcpp.hpp"
```
Comment (threeal) [inline test/compile_test.cpp]: Ditto.

File: test/emptiable_test.cpp [hunk 22]
```diff
@@ -18,8 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#include <gtest/gtest.h>
-#include <shisen_cpp/shisen_cpp.hpp>
+#include "gtest/gtest.h"
```
Comment (threeal) [inline test/emptiable_test.cpp]: Ditto.

**Review by threeal — CHANGES_REQUESTED** [review]

File: src/shisen_cpp/camera/node/camera_node.cpp [hunk 23]
```diff
@@ -24,6 +24,7 @@
 #include <memory>
 #include <nlohmann/json.hpp>
 #include <shisen_cpp/camera/node/camera_node.hpp>
+#include "jitsuyo/config.hpp"
```
Comment (threeal) [inline src/shisen_cpp/camera/node/camera_node.cpp]: Should use angled brackets:
```suggestion
#include <jitsuyo/config.hpp>
```

**Review by threeal — DISMISSED** [review]

**Review by borednuna — DISMISSED** [review]

**Review by FaaizHaikal — APPROVED** [review]

**Review by mbsaloka — APPROVED** [review]

**Pull request merged**